app/models/user.rb

```diff
@@ -1,10 +1,12 @@
 class User < ApplicationRecord
   # Include default devise modules. Others available are:
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable
+  devise :database_authenticatable, :registerable, :recoverable, :rememberable, :trackable,
+         :validatable
+
+  include DeviseTokenAuth::Concerns::User
```

Comment:
Move this include to the first line inside  the class, it's important to keep an order

Reply:
Got it! I moved the include.